frontend/src/ScoreBox.js

```diff
@@ -66,7 +66,6 @@ class ScoreBox extends Component {
     return () => {
       localStorage.setItem('meta-column-type', val ? 'average' : 'recent')
       this.setState({ isAverage: val })
-      this.refs.table.resort()
```

Comment:
@sigmachirality out of curiosity, why is this not necessary anymore?

Reply:
That's a good catch. I just made a few changes moving the isAverage state var from being duplicated in multiple pages to ReviewPage. Lmk what you think! Functional components don't support refs the same way class components do unfortunately...